src/features/reports/components/SendNowModal.tsx

```
@@ -0,0 +1,71 @@
+          </p>
+          <Input
+            label="Recipients (user IDs, comma-separated, optional)"
+            value={recipientsInput}
+            onChange={setRecipientsInput}
+            placeholder="user_123, user_456"
+          />
+        </div>
+      </DialogBody>
+      <DialogFooter>
+        <Button variant="secondary" onClick={onClose} disabled={submitting}>Cancel</Button>
+        <Button onClick={handleSubmit} disabled={submitting}>{submitting ? 'Sending…' : 'Send'}</Button>
```

ESLint

[error] Button text should not change to "Loading...". Keep original label and prepend LoadingSpinner.

(custom/loading-consistency)